### PR TITLE
Prefer github pages in readme

### PR DESCRIPTION
**Pull request by Steveb-p**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,88 +28,88 @@ Features:
...
+    * [STOMP](https://php-enqueue.github.io/transport/stomp/)
 [![Build Status](https://travis-ci.org/php-enqueue/stomp.png?branch=master)](https://travis-ci.org/php-enqueue/stomp)
 [![Total Downloads](https://poser.pugx.org/enqueue/stomp/d/total.png)](https://packagist.org/packages/enqueue/stomp/stats)
 [![Latest Stable Version](https://poser.pugx.org/enqueue/stomp/version.png)](https://packagist.org/packages/enqueue/stomp)
-    * [Amazon SQS](docs/transport/sqs.md)
+    * [Amazon SQS](https://php-enqueue.github.io/transport/sqs/)
 [![Build Status](https://travis-ci.org/php-enqueue/sqs.png?branch=master)](https://travis-ci.org/php-enqueue/sqs)
 [![Total Downloads](https://poser.pugx.org/enqueue/sqs/d/total.png)](https://packagist.org/packages/enqueue/sqs/stats)
 [![Latest Stable Version](https://poser.pugx.org/enqueue/sqs/version.png)](https://packagist.org/packages/enqueue/sqs)
-    * [Amazon SNS](docs/transport/sns.md)
```
Comment (Steveb-p) [inline README.md]: @makasim note, that this file did not actually exist. Should this entry be removed in favor of SNS\SQS?

Comment (makasim) [reply README.md]: fyi https://github.com/php-enqueue/enqueue-dev/issues/769

never had time to add it.

Comment (makasim) [reply README.md]: Could you add an empty file with TODO and a link to the issue?

Comment (Steveb-p) [reply README.md]: @makasim I'm on it.

**Pull request merged**